common/src/misc/utils.ts

```diff
@@ -356,7 +354,7 @@ export class Utils {
     private static getUrlObject(uriString: string): URL {
         try {
             if (nodeURL != null) {
-                return nodeURL.URL ? new nodeURL.URL(uriString) : nodeURL.parse(uriString);
+                return new nodeURL.URL(uriString);
```

Comment:
You're missing the `require` change on line 6 that allows for this change to be the right call.

Reply:
The previous code from https://github.com/bitwarden/jslib/commit/46f9e17056e6248abc4601aede3c2a4c596382c9 was doing `require('url').URL` so that `nodeUrl` was the `URL` object and creating a URL with `return new nodeURL(uriString);`
As I'm calling `return new nodeUrl.URL` I'm creating the new URL object. So it's not necessary to change the require in line 6